host-configs/CVX/cvx-gpu.cmake

```diff
@@ -0,0 +1,64 @@
+site_name(HOST_NAME)
+set(CONFIG_NAME "cvx-gpu" CACHE PATH "" FORCE)
+message( "CONFIG_NAME=${CONFIG_NAME}" )
+
+set(GCC_ROOT "/util/gcc/gcc-9.3.0" CACHE PATH "")
```

Comment:
With this `GCC_ROOT` defined here, you can rely on `cvx-cpu` to define the compilers and you do not have to do it your self. You can overwrite the `fortran` compiler if needed.

Reply:
i created cvx-gcc.cmake and included it in both cvx-cpu.cmake and cvx-gpu.cmake